### DISCUSSION TITLE
Ikemen Go Need a New Look "Stop using elecbyte theme"

### DISCUSSION THREAD

**Discussion by OldGamers** [ikemen-engine/Ikemen-GO#1164] [Engine development]
the mofti is only is 540 kb and has all the goods

you can check the video here
Ikemen GO 99.1 - New Theme Coming soon
https://www.youtube.com/watch?v=WEAAukORi28

I had talk with Windblade-GR01 Gacel.
I asked him very honest question I asked how come you guys are using the theme of Elecbyte?
I also said why cant you guys make your own theme that represent Ikemen go Team ?
only reply was. We not Artist and cant draw as "he clucked and laughed"

 I understand that you guys are trying to demonstrate that the software can demonstrate and use as a fork for the Mugen software and I understand that but we need to stop using the theme because you're confusing people or what the engines is actually capable of doing we understand the most of the text commands are running through the system and everything else similar to Mugen software however everything is running on the zss and also the coded in go language. To demonstrate this is to show you that you need a new look of your software we need to stop using the elecbyte theme of their screen pack and use your own I want to show you a demonstration of the new theme with all the elements that I use was directly from the original source of the Ikemen software the original source that was made by Suehiro and made theme how Suehiro made it for. 

we need to stop supporting elecbyte and let's start supporting the new Ikemen go team with their new look
the screen pack support all resolution in the In-game Option Screen. All sprite are tiled type not static size.
the Motif is only 540kb. all font are in there own and same size for any resolution. 

![ikemen000](https://user-images.githubusercontent.com/35203194/231864940-6380893d-f95b-4850-b237-62861b3a38b4.png)
![0 menu](https://user-images.githubusercontent.com/35203194/231864975-008fa451-9831-4b92-a5be-0099b9ff5c44.png)
![0 option](https://user-images.githubusercontent.com/35203194/231865006-aa1c7633-acae-4126-8b5f-e3b6489caffe.png)
![0 highscore](https://user-images.githubusercontent.com/35203194/231865079-490737b4-5562-4a6c-a366-d1f171588b7e.png)

**Comment by Lazin3ss** [comment]
I think a replacement to elecbyte's motif needs to a have a clean, minimal, friendly design. I find your design proposition to be too bloated with text, and I don't think we should target darker color, more like clearer ones.

**Reply by OldGamers** [reply]
> I think a replacement to elecbyte's motif needs to a have a clean, minimal, friendly design. I find your design proposition to be too bloated with text, and I don't think we should target darker color, more like clearer ones.

friendly base, colorful and simple base noted. I will go back and re do it and thank your for your advice how the motif should be design. thank you for feedback
